src/helpers/kafka.js

```diff
@@ -1,26 +1,34 @@
 // @flow
 import Kafka from 'no-kafka';
-import uuid from 'uuid';
 
 import type { KafkaParams } from '../../types';
 
 const KafkaClient = (params: KafkaParams) => {
```

Comment:
not a big fan of this file, i can see what you're trying to do, but all it does is pass the params into the other constructors.